test/unit/default_responses.js

```diff
@@ -111,7 +95,7 @@ exports['pass filter when message is not from gateway'] = function(test) {
 };
 
 exports['do nothing if reported date is not after config start date'] = function(test) {
-    testUtils.restore([transition._isReportedAfterStartDate]);
+    transition._isReportedAfterStartDate.restore();
```

Comment:
Right, and for the others it's because they are testing that actual function so they need the stub dropped.